src/share/poudriere/jail.sh

```diff
@@ -270,6 +270,7 @@ update_jail() {
 		MASTERNAME=${JAILNAME}-${PTNAME}${SETNAME:+-${SETNAME}}
 		# XXX: Stop doing this (RESOLV_CONF) when freebsd-update -b works
 		[ -n "${RESOLV_CONF}" ] && cp -v "${RESOLV_CONF}" "${JAILMNT}/etc/"
+		[ -n "${PKG_REPO_SIGNING_KEY}" ] && cp -v "${PKG_REPO_SIGNING_KEY}" "${JAILMNT}/${PKG_REPO_SIGNING_KEY%/*}/"
```

Comment:
I think the jail is reverted after all of this but for safety I think we should copy this file in at `update_pkgbase()` and then delete it right after the `make update-packages` call.